bin/scrape_software_versions.py

```diff
@@ -24,6 +24,7 @@
     'circulargenerator':['v_circulargenerator.txt',r"CircularGeneratorv(\S+)"],
     'DeDup':['v_dedup.txt',r"DeDup v(\S+)"],
     'freebayes':['v_freebayes.txt',r"v([0-9]\S+)"],
+    'Sequence Tools':['v_sequencetools.txt',r"v([0-9]\S+)"],
```

Comment:
Make sure that this is picked up in scrape_software.py and the MultiQC module then!